ui/js/components/auth.js

```diff
@@ -38,15 +45,14 @@ treeherder.component("login", {
         // calls to the HTML which sets the user value in the $rootScope.
         onUserChange: "&"
     },
-    controller: ['$location', '$window', 'localStorageService', 'thNotify',
+    controller: ['$location', '$window', 'thNotify',
         'ThUserModel', '$http', 'thUrl', '$timeout', 'thServiceDomain',
-        function ($location, $window, localStorageService, thNotify,
+        function ($location, $window, thNotify,
                   ThUserModel, $http, thUrl, $timeout, thServiceDomain) {
-            var ctrl = this;
-            ctrl.user = {};
-            // "clears out" the user when it is detected to be logged out.
-            var loggedOutUser = { is_staff: false, username: "", email: "", loggedin: false };
+            const authService = new AuthService();
+            const ctrl = this;
 
+            ctrl.user = {};
             ctrl.userLoggingIn = $location.path() === '/login';
```

Comment:
Sorry for missing this before - I think `userLoggingIn` is redundant now that the login callback is on another page? (Which also makes the `<span ng-if=` above redundant)